src/App/App.test.js

```diff
@@ -1,7 +1,106 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
 import App from './App';
+import '@testing-library/jest-dom';
+import { getMovies, checkLoginCredentials } from '../apiCalls';
+import { BrowserRouter } from 'react-router-dom';
+jest.mock('../apiCalls')
 
+describe('App Component', () => {
```

Comment:
Glad we were able to make such great progress with the App testing. Hopefully we can figure out our issues wrt the async testing tomorrow!